ogr/ogrsf_frmts/pg/ogr_pg.h

```diff
@@ -38,6 +38,7 @@
 #include "ogrpgutility.h"
 #include "ogr_pgdump.h"
 
+#include <map>
```

Comment:
Nitpick:
```suggestion
#include <map>
#include <memory>
```
for `std::unique_ptr`.